src/view/treeNodes/fileChangeNode.ts

```diff
@@ -370,7 +371,16 @@ export class GitFileChangeNode extends FileChangeNode implements vscode.TreeItem
 			.sort((a, b) => a.line - b.line);
 
 		if (reviewThreadsForNode.length) {
-			options.selection = new vscode.Range(reviewThreadsForNode[0].line, 0, reviewThreadsForNode[0].line, 0);
```

Comment:
it looks like I used the wrong property here - I wasn't paying attention to the fact that these are outdated review threads, so instead of `line` I think using `originalLine` would also fix this

Though I think there's still a bit of funkiness to whether comments are displayed on commits. This seems to work from the comments panel but not the tree. This should probably be filtered to not just outdated comments, but comments for the specific commit. I think that can be handled separately though

Reply:
Yep, `originalLine` also fixes this. I've removed the re-added code and switched to just use `originalLIne`.